application/controllers/admin/tokens.php

```diff
@@ -902,6 +908,13 @@ function delete($iSurveyID)
             self::_newtokentable($iSurveyID);
         }
 
+        $token = Token::model($iSurveyID)->find('tid=' . $sTokenIDs);
```

Comment:
Fixed. PR made.

Daniel.

2016-05-04 14:58 GMT+02:00 Olle Härstedt <notifications@github.com>:

> In application/controllers/admin/tokens.php
> <https://github.com/LimeSurvey/LimeSurvey/pull/480#discussion_r62033422>:
>
> > @@ -902,6 +908,13 @@ function delete($iSurveyID)
> >              self::_newtokentable($iSurveyID);
> >          }
> >
> > +        $token = Token::model($iSurveyID)->find('tid=' . $sTokenIDs);
>
> This breaks when $sTokenIDs is a list of ids (mark more than one token in
> token list and click delete). Please fix.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/LimeSurvey/LimeSurvey/pull/480/files/71b3dc07093d23e49dd082dcd7609a6703d58d5f#r62033422>
>